src/jit/importer.cpp

```diff
@@ -7401,7 +7401,7 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
 #endif // defined(DEBUG) || defined(INLINE_DATA)
 
                 // Is it an inline candidate?
-                impMarkInlineCandidate(call, exactContextHnd, callInfo);
+                impMarkInlineCandidate(call, exactContextHnd, exactContextNeedsRuntimeLookup == TRUE, callInfo);
```

Comment:
Can you try changing the type of this local to bool (declared on line 6379)?
I don't think that it shoudl be declared a BOOL. 
 (Although the field of exactly the same name in  CORINFO_CALL_INFO is declared a BOOL,
these two aren't assigned into each other)